sphinxcontrib/aafig.py

```diff
@@ -96,12 +98,22 @@ def run(self):
         return [image_node]
 
 
+def warn(app, message):
+    """ Wrapper for warnings emitter to support builders without warn() method
+    """
+    if hasattr(app.builder, 'warn'):
+        app.builder.warn(message)
+    else:
+        warnings.warn(message)
+
+
 def render_aafig_images(app, doctree):
     format_map = app.builder.config.aafig_format
     merge_dict(format_map, DEFAULT_FORMATS)
     if aafigure is None:
-        app.builder.warn('aafigure module not installed, ASCII art images '
-                'will be redered as literal text')
+        message = 'aafigure module not installed, ASCII art images will be' \
+            'rendered as literal text'
+        warn(app, message)
```

Comment:
I would also just put the message in the call instead of creating a `message` variable (same for the other calls to `warn()`), but you are also welcome to leave it as is if you prefer or are just lazy to change it :)